src/sprite_actor.cpp

```diff
@@ -203,14 +203,14 @@ bool Sprite_Actor::IsIdling() {
 
 int Sprite_Actor::GetWidth() const {
 	if (animation) {
-		return animation->GetWidth();
+		return animation->GetAnimationCellWidth() / 2;
 	}
 	return Sprite::GetWidth();
 }
 
 int Sprite_Actor::GetHeight() const {
 	if (animation) {
-		return animation->GetHeight();
+		return animation->GetAnimationCellHeight() / 2;
```

Comment:
RPG Maker 2003 has two types of battle animations (small and big), did you check this?

Reply:
Thanks for the pointer, will make some tests tomorrow with the big sprites. The small ones should work fine.

Reply:
Thanks again for checking on this one, the big sprites were indeed broken. `GetAnimationCellWidth()` and `GetAnimationCellHeight()` always returned 96 (which is correct for normal sprites) but must return 128 on big sprites.